tests/FactoryTest.php

```diff
@@ -66,7 +66,7 @@ public function testSetAlphabet(string $alphabet): void
      *
      * @dataProvider wrongAlphabetsProvider
      */
-    public function testSetWrongAlphabet(string $alphabet): void
+    public function testSetWrongAlphabet($alphabet): void
```

Comment:
can we keep type hint?

Reply:
@garak, this change is fixed automatically via `php-cs-fixer` with customized `.php_cs` setting file.

Perhaps we should disable the rule to let the `@param` doc block and native `string` type be existed at same time :).

Reply:
I'm not aware of any php-cs-fixer rule that is removing type-hinting.
Anyway, it's better to remove phpdoc and keep type-hinting (being the latter a native enforcement)